src/test/unit/scheduler/test_unpack.py

```diff
@@ -27,7 +39,7 @@ def test_unpack_a_container_including_another_container(self, unpacking_schedule
         ]
         unpacking_scheduler.add_task(test_fw)
         extracted_files = {}
-        for _ in range(3):
+        for _ in range(4):
```

Comment:
It seems the test was plainly wrong: 4 objects return from unpacking here (3 files unpacked from the test file) and this test sometimes failed randomly because the order was different and one expected file was missing :facepalm: